Automattic/wp-calypso
# Untangling: show site admin color when in Hosting menus

Related to:

- https://github.com/Automattic/dotcom-forge/issues/6354

## Proposed Changes

This PR will show the site's admin color when we're in one of the Hosting menus. It uses the new endpoint introduced in https://github.com/Automattic/jetpack/pull/36749.

## Testing Instructions

It is important to follow everything EXACTLY.

1. Prepare an early classic Atomic site.
2. Use Jetpack Beta tester to patch https://github.com/Automattic/jetpack/pull/36749.
3. Sandbox public-api.
4. Go to your sandbox.
5. Patch D144492-code to your sandbox (`arc patch D144492`).
6. Patch https://github.com/Automattic/jetpack/pull/36749 to your sandbox (`bin/jetpack-downloader test jetpack untangling/nav-reunification-color`). If asked to checkout to trunk, type `n` to keep in the current branch. When asked to keep the current branch, type `Y`.
8. Go to your site's `/wp-admin/profile.php`.
9. Change the admin color to a color other than the one you set in `/me/account`.
10. Go to `<calypso live URL>/home/<site slug>`.
11. Verify that the color is the same as the color you set in step 8.
12. Now test with an early classic Simple site. You just need to sandbox your site (`<site>.wordpress.com`). You don't need to modify anything in your sandbox again.
13. Verify that this change does not break other sites (not early classic).


## Pre-merge Checklist
